CalculatorWithTests/Calculator.cs

```diff
@@ -1,8 +1,92 @@
-﻿
+﻿using System;
+
 namespace CalculatorWithTests
 {
-    class Calculator
+    public class Calculator
     {
-        
+        public int Addnums(int[] numbers)
+        {
+            int i = 0;
+            foreach (var sum in numbers)
+            {
+                i = i + sum;
+
+            }
+            if (i == 0)
+            {
+                return 0;
+            }
+
+            else
+            {
+                return i;
+            }
+        }
+
+
+        public int Subnums(int[] numbers)
+        {
+            int i = 0;
+            foreach (var sum in numbers)
+            {
+                i = (-sum) + (-i);
+
+            }
+            if (i == 0)
+            {
+                return 0;
+            }
+
+            else
+            {
+                return i;
+            }
+        }
+
+        public int MultiNum(int[] numbers)
+        {
+            int i = 0;
+            foreach (var sum in numbers)
+            {
+                i = (sum) * (i);
+
+            }
+            if (i == 0)
+            {
+                return 0;
+            }
+            else
+            {
+                return i;
+            }
+
+
+        }
+
+        public int DividNum(int[] numbers)
```

Comment:
This is an interesting method that repeatedly divides numbers from an array. I'm not sure what it would be used for, but it looks alright